src/app/modules/main/chat_section/module.nim

```diff
@@ -652,20 +655,17 @@ proc getChatItemFromChatDto(
       memberRole = community.memberRole
 
   var categoryOpened = true
+  let categories = community.categories
 
   if chatDto.categoryId != "":
-    let category = self.controller.getCommunityCategoryDetails(self.controller.getMySectionId(), chatDto.categoryId)
+    let categoryIndex = findIndexById(chatDto.categoryId, categories)
+    let category = categories[categoryIndex]
     if category.id == "":
       error "No category found for chat", chatName=chatDto.name, categoryId=chatDto.categoryId
     else:
       categoryOpened = not category.collapsed
       categoryPosition = category.position
 
-      # TODO: This call will update the model for each category in channels which is not
-      # preferable. Please fix-me in https://github.com/status-im/status-desktop/issues/14431
-      self.view.chatsModel().changeCategoryOpened(category.id, categoryOpened)
```

Comment:
As stated above, this is no longer needed as we set it when creating the item instead